pypesto/visualize/ensemble.py

```diff
@@ -413,3 +414,70 @@ def _create_patches(
         )
 
     return patches_both_hit, patches_lb_hit, patches_ub_hit, patches_none_hit
+
+
+def ensemble_parameters_plot(
+    ensemble: Ensemble,
+    ax: Optional[plt.Axes] = None,
+    parameter_ids: Optional[list[int]] = None,
+    size: Optional[tuple[float]] = (6, 12)
+):
+    """
+    Visualize parameter ensemble.
+
+    Parameters
+    ----------
+    ensemble:
+        ensemble of parameter vectors (from pypesto.ensemble).
```

Comment:
So this is a method to generally just visualize the ensemble, so we cannot really say that the rectangles represent, for example, 95% confidence intervals. But one would need to create an ensemble in some way, and then subset it using a xi^2 threshold to get confidence intervals, right?
Would it be good to have this showcased somewhere? How to use this method correctly? Or is it ok to just expand the description of the visualization in the docstring to describe that it will just visualize all members of the ensemble.